sdk/appcenter/src/main/java/com/microsoft/appcenter/Constants.java

```diff
@@ -24,6 +24,10 @@
      * Maximum number of requests being sent for the group.
      */
     static final int DEFAULT_TRIGGER_MAX_PARALLEL_REQUESTS = 3;
+    /**
```

Comment:
blank line missing before an after

Reply:
I followed the pattern you guys established in this file. Why don't any of the constants above or below have a blank line before and after?

Reply:
sometimes android studio will collapse blank lines by mistake when reordering, I guess it was missed during a prior review

Reply:
Updated this along with the other constants in this file to have lines above and below.